rust-toolchain.toml

```diff
@@ -1,2 +1,4 @@
 [toolchain]
-channel = "nightly"
\ No newline at end of file
+channel = "nightly-2022-10-06"
```

Comment:
This doesn't seem intentional?

Reply:
it is clippy is broken in newer versions and that's how I pin it globally..